index.js

```diff
@@ -35,8 +35,8 @@ async function kubectl() {
 
 async function kind() {
   core.info(`kind`);
-  shell.exec("git clone https://github.com/sustainable-computing-io/KeplerK8SAction --depth=1");
-  shell.exec("cd KeplerK8SAction && bash -c ./kind/common.sh")
+  shell.exec("git clone https://github.com/sustainable-computing-io/local-dev-cluster.git --depth=1");
+  shell.exec("cd local-dev-cluster && bash -c ./mian.sh")
```

Comment:
@rootfs , @marceloamaral if we use https://github.com/sustainable-computing-io/local-dev-cluster to save our local env set up code, but not this repo. this line will skip testing... as the code is out of CI scope.
if we want to cover into CI testing, https://github.com/sustainable-computing-io/kepler-action/pull/19/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R39 this PR is a sample.

Reply:
Looks good, let's go this path!